client/signup/steps/site-type/index.jsx

```diff
@@ -109,6 +118,17 @@ export default connect(
 	( dispatch, ownProps ) => ( {
 		submitStep: ( siteTypeValue, themeRepo ) => {
 			dispatch( setSiteType( siteTypeValue ) );
```

Comment:
@niranjan-uma-shankar: can we add an explicit `calypso_signup_actions_submit_site_type` tracks event here, like [we ended up doing for siteTopic](https://github.com/Automattic/wp-calypso/blob/master/client/signup/steps/site-topic/index.jsx#L156)? In our funnel audit, I think it would make things clearer.

Reply:
Fixed in [bf39dc3](https://github.com/Automattic/wp-calypso/pull/28672/commits/bf39dc38849da0a1424461f60a5286740827f9bd)